crates/flox/src/commands/mod.rs

```diff
@@ -446,88 +448,130 @@ pub enum EnvironmentSelect {
 }
 
 impl EnvironmentSelect {
-    /// Open a concrete environment, optionally detecting the currently active
+    /// Open a concrete environment, not detecting the currently active
```

Comment:
Suggestion: It's not clear to me when you would want to use `to_concrete_environment` vs. `detect_concrete_environment` based on this description. Could you add a little bit more explanation?

Reply:
Added